src/firefly/js/core/background/JobInfo.jsx

```diff
@@ -83,8 +80,8 @@ function DataOrigin({dataOrigin, type, jobId}) {
     if (!isUws) return <KeywordBlock label={label} value={dataOrigin} asLink={true}/>;
     return (
         <div style={{display: 'inline-flex'}}>
-            <KeywordBlock style={{width: 425, alignItems:'center'}} label={label} value={dataOrigin} asLink={true}/>
-            <Button ml={1/2} onClick={() => showUwsJob({jobId})}>Show</Button>
```

Comment:
Should I also convert this to Chip for consistency with Table metadata tab?

Reply:
I think the uppercase "Show" fits better with the style of the Job Info tab, and the lowercase one fits better with the style of the Metadata tab, where in general everything is lowercase.  So I don't think it needs to be changed.

Incidentally, the text baselines are perfectly aligned in the Job Info version of this:

<img width="907" alt="image" src="https://github.com/Caltech-IPAC/firefly/assets/13261203/a4c5ccb0-d5d4-4294-bd74-c17244009348">